libnest/src/config.rs

```diff
@@ -59,10 +63,26 @@ impl Config {
     /// ```
     #[inline]
     pub fn new() -> Config {
-        Config {
-            cache: PathBuf::from("/var/lib/nest/cache/"),
-            download_path: PathBuf::from("/var/lib/nest/download/"),
-            repositories: Vec::new(),
+        Config::default()
+    }
+
+    /// Loads a configuration from a TOML file
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # extern crate libnest;
+    /// use libnest::config::Config;
+    ///
+    /// let mut config = Config::new();
+    /// config.load_conf("Config.toml");
+    /// ```
+    pub fn load_conf(&mut self, path: &str) {
+        match ConfigParser::new(path) {
+            Ok(conf_parser) => {
```

Comment:
Brackets are un-needed if there is only one item
`Ok(conf_parser) => conf_parser.load_to_config(self),`